### DISCUSSION TITLE
TimeoutUser

### DISCUSSION THREAD

**Discussion by Tqeslib** [Sehelitar/Kick.bot#66] [Q&A]
Hello, its me again. I have a few questions. Firstly why is this not working? Where am i doing wrong?

![timeoutuser](https://github.com/user-attachments/assets/e721efd1-54b8-42bf-8e21-0ae70f8edf3f)

Secondly, how can I run it on Kick?

![Screenshot_4](https://github.com/user-attachments/assets/f8f1abd8-05e4-43d1-89fa-bac9b4344a56)
![Screenshot_6](https://github.com/user-attachments/assets/65ddc148-4fbf-42cd-9186-c62b0dee0dce)

**Comment by Sehelitar** [comment]
Hi! 👋 

> Hello, its me again. I have a few questions. Firstly why is this not working? Where am i doing wrong?
> 
> ![timeoutuser](https://private-user-images.githubusercontent.com/176499284/352779796-e721efd1-54b8-42bf-8e21-0ae70f8edf3f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIxODQxNjEsIm5iZiI6MTcyMjE4Mzg2MSwicGF0aCI6Ii8xNzY0OTkyODQvMzUyNzc5Nzk2LWU3MjFlZmQxLTU0YjgtNDJiZi04ZTIxLTBhZTcwZjhlZGYzZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyOFQxNjI0MjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xOGIzODY5ZTdhZGY3OTcwOGE5YTlkZWRlNWJiNDFmZGZkYjg5MDg5NDQwMjgyMjA4MzE2YmM3NWIyNjk5N2RlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.YcGQkpl0y3r3Y45WekN-KldcrHMLyg2CFhu4-znnaps)

You should have 1 argument per line before the call to the method.
Ex: 
![image](https://github.com/user-attachments/assets/48e5be99-3428-4295-9c38-497b84c1b22b)


> Secondly, how can I run it on Kick?
> 
> ![Screenshot_4](https://private-user-images.githubusercontent.com/176499284/352780153-f8f1abd8-05e4-43d1-89fa-bac9b4344a56.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIxODQxNjEsIm5iZiI6MTcyMjE4Mzg2MSwicGF0aCI6Ii8xNzY0OTkyODQvMzUyNzgwMTUzLWY4ZjFhYmQ4LTA1ZTQtNDNkMS04OWZhLWJhYzliNDM0NGE1Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyOFQxNjI0MjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zZDY5ZjIyYjkwZTgyOTBmNTEzZDUyNTY5MTQ4ZDRlZjY1ODhlNGY0NDg2NjFlMGFhNjk2Yzk4NGRjZTgzOTVhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.fM-C9ELI_U5IX6PRJHTIYbDZ97H5ciF6vISB2YVsx6U) ![Screenshot_6](https://private-user-images.githubusercontent.com/176499284/352780157-65ddc148-4fbf-42cd-9186-c62b0dee0dce.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIxODQxNjEsIm5iZiI6MTcyMjE4Mzg2MSwicGF0aCI6Ii8xNzY0OTkyODQvMzUyNzgwMTU3LTY1ZGRjMTQ4LTRmYmYtNDJjZC05MTg2LWM2MmIwZGVlMGRjZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyOFQxNjI0MjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xOGQ0MmI0YzhiZjMwYmNlZTk5ZmY3NWEyMzVkMTFkZTY0ZDI0NDUyMjRlZDlhMjMzNWE0ZmVlM2Y0OTQ3NzQ5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Gr6f1DL8eWB5H9oWYspkiJaG9BspLadh9j2Yf6obDws)

Replace your CPH.SendMessage with CPH.SetArgument and set the ``reply`` argument with your translated message.
You won't need the "Set argument" sub-action before the call to SendReply anymore, so you can remove that.

**Comment by Tqeslib** [comment]
Hello its me again. Thanks for your advice. But when i compile the code it shows me that errors. How can i fix them? I tried running the command by reintroducing kick.dll and kickbot.dll. Result is same. Thanks for your answer in advance.
![Screenshot_8](https://github.com/user-attachments/assets/9e62c963-f104-4f0c-ba1c-c99aeda72cfc)
![Screenshot_9](https://github.com/user-attachments/assets/ea61f1cd-e682-4fd5-bec9-0caeffd09ebc)

**Reply by Sehelitar** [reply]
Hi! 👋 

Line 13 should be : 
```C#
CPH.SetArgument("reply", Translate(text, fromLanguage, toLanguage));
```
